pkg/database/vercode_test.go

```diff
@@ -447,7 +447,7 @@ func TestStatDates(t *testing.T) {
 	db, _ := testDatabaseInstance.NewDatabase(t, nil)
 	realm := NewRealmWithDefaults("Test Realm")
 
-	now := time.Now()
+	now := time.Now().UTC()
```

Comment:
This was a bug. This test fails in the evening.

Reply:
Nice. I've hit a few of those when I run tests late at night too.